src/obs-controller.cpp

```diff
@@ -391,13 +391,13 @@ void ToggleSourceFilter(MidiHook *hook)
 		obs_source_set_enabled(filter, true);
 	}
 }
-void TriggerHotkeyByName(MidiHook *hook)
+void TriggerHotkey(MidiHook *hook)
 {
-	obs_hotkey_t *hk = Utils::FindHotkeyByName(hook->hotkey);
-	if (!hk) {
+	obs_hotkey_t *hotkeyInstance = hook->getHotkey();
+	if (!hotkeyInstance) {
```

Comment:
do not throw errors, use Utils::alert_popup

Reply:
and then return;

Reply:
Ok. `throw` was there like forever :D 
I didn't know how to inform user, so just kept the old code.
Gonna change it